shop-guide/src/utilities.ts

```diff
@@ -127,10 +127,11 @@ export class ShopUtilities {
 
   @PostApi('/payment_webhook')
   static async paymentWebhook(ctxt: HandlerContext): Promise<void> {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
```

Comment:
I wonder if we are missing devDependency on `@types/koa` or something like that.  You can do this in a prettier way for sure, none of the other demos have this issue.

Reply:
Resolved this - we were missing the line in devDependencies